src/net/tcp/socket.rs

```diff
@@ -82,6 +82,16 @@ impl TcpSocket {
         sys::tcp::set_reuseaddr(self.sys, reuseaddr)
     }
 
+    /// Sets the value of `SO_REUSEPORT` on this socket.
```

Comment:
Can you add `pub fn reuse_port(&self) -> io::Result<bool>` to retrieve the value?